jest.config.js

```diff
@@ -8,8 +8,11 @@ module.exports = {
   transform: {
     '^.+\\.vue$': 'vue-jest',
     '.+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$': 'jest-transform-stub',
-    '^.+\\.jsx?$': 'babel-jest',
+    '^.+\\.(js|jsx)?$': '<rootDir>/node_modules/babel-jest',
```

Comment:
Why did you change this line ? I tried old regex and path and it works.

Reply:
When I was writing tests, It did not work on my environment. I changed pc and install all dependencies again, now It is working without `<rootDir>/node_modules`. :)